test/suites/ubuntu-core-edgex-image/build.sh

```diff
@@ -1,47 +1,70 @@
 #!/bin/bash -ex
 
+# Remove the pc-gadget directory if it already exists
 rm -rf pc-gadget
 git clone https://github.com/snapcore/pc-gadget.git --branch=22
 
-# build gadget snap
+# Build gadget snap
 cd pc-gadget
-# extend the size of disk partitions to have sufficient capacity for EdgeX snaps
-yq e '(.volumes.pc.structure[] | select(.name=="ubuntu-seed") | .size)="1500M"' gadget.yaml -i
 
-# setup default options for snaps
+# Extend the size of disk partitions to have sufficient capacity for EdgeX snaps
+yq e -i '(.volumes.pc.structure[] | select(.name=="ubuntu-seed") | .size)="1500M"' gadget.yaml
+
+# Set up default options for snaps
 # AZGf0KNnh8aqdkbGATNuRuxnt1GNRKkV (edgexfoundry snap)
 # AmKuVTOfsN0uEKsyJG34M8CaMfnIqxc0 (edgex-device-virtual snap) 
-yq e '.defaults += {
+yq e -i '.defaults += {
   "AZGf0KNnh8aqdkbGATNuRuxnt1GNRKkV": {
     "app-options": true,
-    "security": false
+    "security-secret-store": "off"
   },
   "AmKuVTOfsN0uEKsyJG34M8CaMfnIqxc0": {
     "autostart": true,
-    "app-options": true,
     "apps": {
       "device-virtual": {
         "config": {
-          "service-startupmsg": "Startup message from gadget!",
           "edgex-security-secret-store": false
```

Comment:
It seems like the changes didn't take effect. Based on the logs, device-virtual is still searching for the token:
```
[   57.934013] edgex-device-virtual.device-virtual[3550]: level=INFO ts=2023-04-04T09:49:49.694645244Z app=device-virtual source=secret.go:177 msg="load token from file"
[   57.934061] edgex-device-virtual.device-virtual[3550]: level=WARN ts=2023-04-04T09:49:49.694668645Z app=device-virtual source=secret.go:119 msg="Retryable failure while creating SecretClient: open /var/snap/edgex-device-virtual/335/device-virtual/secrets-token.json: no such file or directory"
```

Reply:
This known config option issue has been resolved by [this](https://github.com/edgexfoundry/device-virtual-go/pull/354) PR in the latest/edge channel of device-virtual. Running this test with device-virtual from the latest/edge channel should produce the expected results.